test/powershell/Modules/Microsoft.PowerShell.Utility/Remove-Alias.Tests.ps1

```diff
@@ -0,0 +1,64 @@
+
+#Store ErrorActionPreference Value
+$storedEA = $ErrorActionPreference
+$ErrorActionPreference = "Stop"
+Describe "Remove-Alias" -Tags "CI" {
+
+    It "Remove-Alias Should Remove Non-ReadOnly Alias"{
```

Comment:
Space skipped.
And please use normal case for normal words:
`"Remove-Alias should remove a non-readonly alias" {`

Below too.